src/common/components/help/install-snap/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component, PropTypes } from 'react';
+
+import { HeadingSix } from '../../vanilla/heading/';
```

Comment:
They could be. When they're as small and as simple as this, the difference seems marginal.